.env.example

```diff
@@ -8,3 +8,5 @@ POSTGRES_PASSWORD=laguan
 POSTGRES_DB_URL=0.0.0.0
 POSTGRES_DB=postgres
 DATABASE_URL=postgres://postgres@localhost/../error-microservice/
+
+CORS_ORIGIN=localhost:8080
```

Comment:
Notice that in server configuration there is a whitelist field  please use that one instead

Reply:
Good catch!. I changed it for WHITELIST, please check if is ok now